src/query/storage/completed_tags.go

```diff
@@ -81,10 +81,13 @@ func (s completedTagsByName) Less(i, j int) bool {
 }
 
 func (b *completeTagsResultBuilder) Build() CompleteTagsResult {
-	result := make([]CompletedTag, len(b.tagBuilders))
+	result := make([]CompletedTag, 0, len(b.tagBuilders))
 	if b.nameOnly {
 		for name := range b.tagBuilders {
-			result = append(result, CompletedTag{Name: []byte(name)})
```

Comment:
oops, did it break any test now ?

Reply:
This was only added in this PR; once tests were added it failed a few